src/js/game/LevelMVC/FacingDirection.js

```diff
@@ -13,6 +13,14 @@ const FacingDirection = Object.freeze({
     }
   },
 
+  left: function (facing) {
+    return (facing + 3) % 4;
+  },
+
+  right: function (facing) {
+    return (facing + 1) % 4;
+  },
+
```

Comment:
nit; there's a bit of overlap with these and the `turn(facing, rotation)` method; it'd be nice to update one of em to use the other.